src/server/adyenMiddleware.ts

```diff
@@ -0,0 +1,54 @@
+import Axios from 'axios'
+import * as Koa from 'koa'
+import { GIRAFFE_ENDPOINT } from 'server/config'
+import { createSession, Session } from 'utils/sessionStorage'
+import { ServerCookieStorage } from 'utils/storage/ServerCookieStorage'
+
+interface Adyen3dsDetails {
+  MD: string
+  PaRes: string
+}
+
+const httpClient = Axios.create({
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+  },
+})
+
+// FIXME use this
```

Comment:
fixme

Reply:
some, 3d secure. Probably needed. I'd rather have the fixme than forget about it :D